### ISSUE TITLE
Question about classes in GENIA 

### ISSUE THREAD

**Issue by zhangchengxian-git** [juntaoy/biaffine-ner#48]
Sorry to bother you, but i have a question about GENIA in your code. Do you consider GENIA's categories as the following six categories?
<img width="378" alt="image" src="https://user-images.githubusercontent.com/57203872/220676463-43caf035-71c3-4e58-b864-744d89d584c0.png">
If so, does your method perform well in fine-grained division (47 categories) in GENIA?

**Comment by juntaoy** [comment]
Yes, it only evaluated the coarse-grained setting I haven’t tested it on the fine-grained version. It does very well on Ontonotes which consist of 18 NER labels, I assume it might also work on the 47 labels, but it might depend on the training data size.

**Comment by zhangchengxian-git** [comment]
I got it! Thank you so much.

**Comment by liuwb2001** [comment]
How do you get the "char_vocab.eng.genia.txt"
